doc/sphinx/docs/python/tutorials/evolving_a_population.rst

```diff
@@ -0,0 +1,79 @@
+.. py_tutorial_evolving_population
+
+Evolving a population
+=====================
+
+Solving an optimization problem using an optimization algorithm is, in pygmo,
+described as *evolving* a population. In the scientific literature, an interesting
+discussion has developed over the past decades on whether evolution is or not a form of
+optimization. In pygmo we take the opposite standpoint and we regard optimization,
+of all types, as a form of evolution. Regardless on whether you will be using an SQP,
+an interior point optimizer or an evolutionary startegy solver, in pygmo you will
+always have to call a method called ``evolve`` to improve over your initial solutions,
```

Comment:
Can we link here to the ``evolve()`` method of the algorithm class? We should always use the trailing brackets ``()`` when referring to a function/method, so it is clear that we are referring to a callable entity.